lib/components/PoolData.jsx

```diff
@@ -127,7 +127,7 @@ const UnsupportedNetwork = (props) => {
         {SUPPORTED_NETWORKS.map((network) => {
           if ([31337, 1234].includes(network)) return null
           return (
-            <li className='ml-2 xs:ml-4'>
+            <li className='ml-2 xs:ml-4' key='network'>
```

Comment:
Keys need to be unique, so each row has it's own key. If network has an id, then 

```js
key={`network-${network.id}`}
```